.github/workflows/mgfield.yml

```diff
@@ -0,0 +1,91 @@
+name: Build Test on different OSs
+
+on:
```

Comment:
Seems you forgot to change the branches, maybe make it run on all branches instead
Something like
`on: [push, pull_request]`

Reply:
Done